packages/pyo3_samples/pyo3_config.ini

```diff
@@ -0,0 +1,9 @@
+implementation=CPython
+version=3.9
+shared=true
+abi3=false
+lib_name=python3.9
+lib_dir=/home/hood/Documents/programming/pyodide/cpython/installs/python-3.9.5/lib/
```

Comment:
It's not used then if it works in CI?

Reply:
Apparently not. I am pretty unclear on which of these are used -- without this file it definitely doesn't work, but spme of the entries seem to have no effect.